src/js/gallery.js

```diff
@@ -21,12 +21,22 @@ const searchParams = () =>
     page: page,
   });
 
+observerGalleryEl.style.display = 'none';
+
+let wasFirstFech = false;
```

Comment:
Tu jest chyba literówka w słowie "Fetch". Jeśli zmienna będzie jeszcze gdzieś wykorzystywana, to może powodować problemy i "głupie" błędy :)